source/telemetry/telemetry_logging/logging_configuration.py

```diff
@@ -63,16 +115,31 @@ def configure_logging(
         _EXTRAS = extras.copy()
 
     # Add cloud role name when logging
-    os.environ["OTEL_SERVICE_NAME"] = cloud_role_name
+    os.environ["OTEL_SERVICE_NAME"] = logging_settings.cloud_role_name
 
     # Configure OpenTelemetry to log to Azure Monitor.
-    if applicationinsights_connection_string is not None:
-        configure_azure_monitor(connection_string=applicationinsights_connection_string)
+    if logging_settings.applicationinsights_connection_string is not None:
+        configure_azure_monitor(
+            connection_string=logging_settings.applicationinsights_connection_string
+        )
         _IS_INSTRUMENTED = True
 
     # Reduce Py4J logging. py4j logs a lot of information.
     logging.getLogger("py4j").setLevel(logging.WARNING)
 
+    # Adding orchestration ID as an extra when provided through LoggingSettings: This is required for the Telemetry logs to show up in Application Insights
```

Comment:
"This is required for the Telemetry logs to show up in Application Insights" - is this true?